cypress/integration/WebInterface/Test Cases/Validations/StratificationExpectedValues.spec.ts

```diff
@@ -523,6 +523,11 @@ describe('Stratification Expected values for Non Boolean Population Basis', () =
         cy.get(TestCasesPage.runTestButton).should('be.enabled')
         cy.get(TestCasesPage.runTestButton).click()
 
+        //validate color of Measure Group-1 Stratification after running
```

Comment:
I would just drop this check, its a bit too granular for us for now. And we know it is going to be changing soon

Reply:
Ok.
I will remove this